packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -30,6 +30,7 @@
     <checkstyle.skip>true</checkstyle.skip>
     <notice.dir>${project.basedir}/src/main/resources/META-INF</notice.dir>
     <notice.file>HUDI_NOTICE.txt</notice.file>
+    <avro.scope>provided</avro.scope>
```

Comment:
nit: rename to `mr.bundle.avro.scope` ? in case, we decide this strategy for other bundles too?